src/index.mako.html

```diff
@@ -173,7 +173,7 @@
           <a href="" ng-click="globals.feedbackPopupShown = !globals.feedbackPopupShown">
             <span ng-class="{'selected': globals.feedbackPopupShown}" translate>problem_announcement</span>
           </a>&nbsp;&nbsp;
-          <a target="_blank" ng-href="{{topicId + '_help_link_href' | translate}}" translate>help_label</a>&nbsp;&nbsp;
+          <a target="_blank" ng-href="//help.geo.admin.ch/?lang={{langId}}" translate>help_label</a>&nbsp;&nbsp;
```

Comment:
Really? But if you use a std href, {{langId}} won't be interpreted correctly! No?